java/HealthVault Mobile/HVSample/src/main/res/layout/weight.xml

```diff
@@ -4,16 +4,28 @@
 	android:layout_height="match_parent">
 
 	<LinearLayout
+		android:id="@+id/graphContainer"
+		android:layout_width="match_parent"
+		android:layout_height="237dp"
+		android:background="#008272">
+		<com.jjoe64.graphview.GraphView
+			android:layout_width="match_parent"
+			android:layout_height="match_parent"
+			android:id="@+id/graph" />
+	</LinearLayout>
+
+	<LinearLayout
```

Comment:
Looks like we can just get rid of the root level RelativeLayout, all its children are vertically stacked making it act just as a linear layout would. Seems like we can just have one root level LinearLayout with a vertical orientation and then the graph, add weight, last weight, labels etc will just all be stacked vertically. This should simplify the layout a lot.